search/templates/search/search.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Maybe extend the base template @nadavsu made in main? we will have to do it anyways

Reply:
In this PR I just created the `search` App. 
I will upgrade the app (including the html file) in the next PRs, so in order to keep the PR  simple, I chose to create it without the base template.

Anyway, I just added that as you asked. 
Done :)